app/controllers/admin/items_controller.rb

```diff
@@ -0,0 +1,15 @@
+class Admin::ItemsController < ApplicationController
```

Comment:
If we are namespacing it in the items controller, can guests still see the menu?